mappings/net/minecraft/network/packet/c2s/play/RecipeBookDataC2SPacket.mapping

```diff
@@ -11,6 +11,12 @@ CLASS net/minecraft/class_2853 net/minecraft/network/packet/c2s/play/RecipeBookD
 	FIELD field_17206 smokerGuiFilteringCraftable Z
 	METHOD <init> (Lnet/minecraft/class_1860;)V
 		ARG 1 recipe
+	METHOD <init> (ZZZZZZ)V
+		ARG 1 guiOpen
+		ARG 2 filteringCraftable
+		ARG 3 furnaceGuiOpen
+		ARG 4 furnaceFilteringCraftable
+		ARG 5 blastFurnaceGuiOpen
```

Comment:
You could have done the last parameter at the same time :)

Reply:
That last parameter used to set 2 fields of the packet (blastFurnaceFilteringCraftable, smokerGuiFilteringCraftable) and they are being send separately. Weird flex, but ok. I don't have any good names on my mind.
Only one stuff unites both entity blocks: they are used to smelt 2 times faster. 😕

Reply:
`filteringCraftable`

Reply:
@liach there is `filteringCraftable` already

Reply:
then `cookingFilteringCraftable` I think if that applies to blast furnace, furnace, and that 4 log one furnace food smelter

Reply:
@liach Furnace is specified with `furnaceFilteringCraftable` as can be seen in `ARG 4`, but `cookingFilteringCraftable` is better than nothing.

Reply:
Please rename arg 5 as well to match arg 6 ;)